amethyst_animation/src/bundle.rs

```diff
@@ -1,9 +1,13 @@
-use amethyst_core::specs::prelude::{Component, DispatcherBuilder};
-use amethyst_core::{Result, SystemBundle};
+use std::{hash::Hash, marker};
+
 use resources::AnimationSampling;
 use skinning::VertexSkinningSystem;
-use std::hash::Hash;
-use std::marker;
+
+use amethyst_core::{
+    specs::prelude::{Component, DispatcherBuilder},
+    {Result, SystemBundle},
```

Comment:
No need to group those (also doesn't look very good).

Reply:
ah yeah, forgot to remove the `{}` around Result and SystemBundle.
Other than that, why not group those?

Reply:
Because they are part of the outer grouping

Reply:
So:
```rust
use amethyst_core::specs::prelude::{Component, DispatcherBuilder};
use amethyst_core::{Result, SystemBundle};
```?

Reply:
```rust
use amethyst_core::{
    specs::prelude::{Component, DispatcherBuilder},
    Result,
    SystemBundle,
};
```

Very simple, sorry for being unclear.